GeneratingDataset.py

```diff
@@ -2718,13 +2748,13 @@ def get_tag(self, seq_idx):
     :param int seq_idx:
     :rtype: str
     """
-    return self._data[self._get_ref_seq_idx(seq_idx)]["file"]
+    return self._data[self._get_ref_seq_idx(seq_idx)]["seq_name"]
```

Comment:
It should stay compatible to existing datasets. I.e. try to read "seq_name", but if it is not there, then fall back to "file".

Reply:
Sorry, I did not saw that you loop over the whole dataset, and fix that dict. But as written there, better don't do that, and just do it on-the-fly here.